dGame/dGameMessages/GameMessages.h

```diff
@@ -1,5 +1,4 @@
-#ifndef GAMEMESSAGES_H
```

Comment:
This traditional include guard must stay.  It is in PCH files and pragma once is not standard.